lib/src/git.rs

```diff
@@ -117,6 +121,24 @@ pub fn import_refs(
             }
         }
     }
+
+    // Find commits that are no longer referenced in the git repo and abandon them
+    // in jj as well.
+    let old_git_heads = old_git_heads.into_iter().collect_vec();
+    let new_git_heads = new_git_heads.into_iter().collect_vec();
+    // We could use mut_repo.record_rewrites() here but we know we only need to care
+    // about abandoned commits for now. We may want to change this if we ever
+    // add a way of preserving change IDs across rewrites by `git` (e.g. by
+    // putting them in the commit message).
+    let abandoned_commits = mut_repo
+        .index()
+        .walk_revs(&old_git_heads, &new_git_heads)
```

Comment:
Is there a way of using named parameters in rust, or some canonical mechanism to specify the name of the parameter in a way that linters would check? Having this be something like .walk_revs(wanted=&old_git_heads, unwanted=&new_git_heads)  would have made this a lot more readable for me.

Reply:
I agree, but I think we'll have to wait for https://github.com/rust-lang/rfcs/issues/323, if that ever happens. (My IDE shows the argument names if they don't match the variable names, but that doesn't help during review etc.)